index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Aprendiendo Flexbox</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header>
+        <div class="boton">≡</div>
+    </header>
+    <section class="cuerpo">
+        <h1 class="caja miNombre">Felipe Haybar</h1>
+        <section class="saludo">
+            <p>
+            Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works. If you like what you see and have a project you need coded, don’t hesitate and contact me.
+            </p>
+         </section>    
+    </section>
+    <section class="collabBoton"><p> start colaboration </p></section>
+    <div class="art"><img src="images/art.png" alt="artistic forms" width="320px"></div>
+    <section class="projects">
+        <h4>Projects</h4>
+        <img src="images/project1.png" alt="project1" width="320px">
```

Comment:
**OPTIONAL** - As the previous reviewers have suggested, all images that aren't part of your page's content should be added using the `background` property in CSS. Please fix this for all images as they may apply 👍 👍